.github/workflows/main.yml

```diff
@@ -384,7 +384,9 @@ jobs:
 
   delete-test-images:
     runs-on: ubuntu-latest
-    needs: test
+    needs:
+      - test
+      - build
```

Comment:
is this really necessary? if test is already depending on build there is no change in behaviour with this change

Reply:
Without this the env variables added at https://github.com/Activiti/activiti-cloud-application/pull/694/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R398 that depend on the "build" job are not filled correctly, see https://github.com/Activiti/activiti-cloud-application/runs/7500000853?check_suite_focus=true (PREVIEW_NAME, BRANCH_NAME and VERSION are empty)

Reply:
And I agree it's not obvious, so I can add a comment to explain this

Reply:
oh and now I remember stumbling on it at least twice in the past :D